### PR TITLE
Fix for #482 - Removed Next button and overlay

### PR DESCRIPTION
**Pull request by theresa-lee**
Not sure what emoji to add but can do that in a separate PR! I just removed the button and overlay for now.

### CODE REVIEW

**Comment by rabidbit** [comment]
Looks like the "Check my code" and "I think I'm done" buttons are back to being blue?

File: client/question/components/LearnerViewDirective.js [hunk 1]
```diff
@@ -408,18 +355,16 @@ tie.directive('learnerView', [function() {
...
           position: relative;
         }
         .tie-run-button:hover {
           box-shadow: inset 0 1px 2px rgba(0,0,0.3);
         }
-        .tie-run-button:active {
-          background-color: rgb(42, 112, 232);
-          border: 1px solid rgb(32, 100, 200);
-          box-shadow: inset 0 1px 2px rgba(0,0,0.3);
-        }
```
Comment (rabidbit) [inline client/question/components/LearnerViewDirective.js]: I think it's OK to have an :active rule, to distinguish it from hover over. Usually, a darker shade of the same color is used though (I think we kept the dark blue color from prior versions, though I'm not certain that was intentional--darker green for example would make sense for "I think I''m done".

Comment (theresa-lee) [reply client/question/components/LearnerViewDirective.js]: Whoops, sorry, I think I got a little aggressive with removing "dead" code but I put it back now! I actually made some changes to the button hovering (including what happens in dark mode), but kept the general idea of it getting darker for hover and then even a bit darker for active. The dark mode has only the border highlighted on hover (similar to the reset and theme buttons in dark mode) but also turns a bit darker on press. I am happy to change color/affect if anyone has objections.

Comment (theresa-lee) [reply client/question/components/LearnerViewDirective.js]: I split this part up into their respective color CSS (since we were using blue for both).

**Comment by rabidbit** [comment]
One last comment, otherwise LGTM. The buttons are now brighter in light mode, whereas we intentionally made them more muted to keep the user's attention on the code. In particular, the blue on the "Check my code" stands out more. Can we keep the more muted colors for now?

Thanks for the fix!

**Pull request merged**